src/archiefvernietigingscomponent/destruction/models.py

```diff
@@ -200,6 +200,29 @@ def __str__(self):
         return f"{self.destruction_list}: {self.author}"
 
 
+class DestructionListReviewComment(models.Model):
+    review = models.ForeignKey(
+        DestructionListReview,
+        on_delete=models.CASCADE,
+        related_name="comments",
+        verbose_name=_("destruction list review"),
+    )
+    text = models.TextField(
+        _("text"),
+        max_length=2000,
+        blank=True,
+        help_text=_("Text of the comment to the review"),
+    )
+    created = models.DateTimeField(_("created"), default=timezone.now)
+
+    class Meta:
+        verbose_name = _("destruction list review comment")
```

Comment:
Sorry I missed it in the first review, but I think that `verbose_name` should be added to `django.po` file, now it stands out in the admin

Reply:
I updated the translation files!